dashboard/app/controllers/projects_controller.rb

```diff
@@ -67,11 +67,14 @@ def load
 
   def create_new
     return if redirect_under_13(@level)
-    redirect_to action: 'edit', channel_id: create_channel({
-      name: 'Untitled Project',
-      useFirebase: use_firebase_for_new_project?,
-      level: polymorphic_url([params[:key], 'project_projects'])
-    })
+    redirect_to action: 'edit', channel_id: ChannelToken.create_channel(
+      request.ip,
+      StorageApps.new(storage_id('user')),
+      {
+        name: 'Untitled Project',
+        useFirebase: @level.game.use_firebase_for_new_project?,
```

Comment:
Nit: might be better to merge this in `create_channel` if it's always `@level.game.use_firebase_for_new_project?` when data is present.